types/googlemaps/test/reference/event-tests.ts

```diff
@@ -0,0 +1,15 @@
+import event = google.maps.event;
+
+event.addDomListener({}, 'click', (e: MouseEvent) => {
+    e; // $ExpectType MouseEvent
+});
+
+event.addDomListener({}, 'keydown', (e: KeyboardEvent) => {
+    e; // $ExpectType KeyboardEvent
+});
+
+event.addDomListener({}, 'load', (e: Event) => {
+    e; // $ExpectType Event
+});
```

Comment:
You're explicitly typing the event param instead of letting it be inferred by the event name. Therefore these tests would pass as long as the explicit type is compatible with the implicit one.

```suggestion
event.addDomListener({}, 'click', (e) => {
    e; // $ExpectType MouseEvent
});

event.addDomListener({}, 'keydown', (e) => {
    e; // $ExpectType KeyboardEvent
});

event.addDomListener({}, 'load', (e) => {
    e; // $ExpectType Event
});
```

Reply:
Thanks for that suggestion. Yep that makes sense. I've made the changes you've suggested and pushed them up.